mscore/scoreview.cpp

```diff
@@ -1961,6 +1961,18 @@ void ScoreView::cmd(const QAction* a)
             else
                   cmdGotoElement(score()->lastElement());
             }
+      else if (cmd == "next-palette-element") {
+            MuseScore* ms = static_cast<MuseScore*>(this->parent());
```

Comment:
mscore->getPaletteBox(); should be enough?
The rest could be put in a new method? PaletteBox::nextPaletteElement?